pkg/util/intstr/intstr.go

```diff
@@ -101,6 +105,17 @@ func (intstr IntOrString) MarshalJSON() ([]byte, error) {
 	}
 }
 
+func (_ IntOrString) OpenAPIDefinition() common.OpenAPIDefinition {
```

Comment:
This seems really wrong - now we're coupling every Go type that ever uses IntOrString with commonapi?  

This really needs to be outside this type as a separate package.